src/site/_includes/layouts/markdown-content-page.njk

```diff
@@ -7,7 +7,7 @@
   <div class="guidance-split-pane__nav">
     {{ guidanceSideNav({
       url: page.url,
-      items: collections[tags[0]] | sort(false, false, 'data.order')
+      items: collections[tags[0]] | rejectattr_path('data.is_root') | sort(false, false, 'data.order')
```

Comment:
Looks good, I probably need the inverse of this that only includes the root for the primary nav

Reply:
Yeah I thought I left a comment but the built-in `rejectattr`/`selectattr` don't support dotted paths.